src/utils/userName.ts

```
@@ -0,0 +1,7 @@
+import api from '../lib/axios';
+
+export async function getUserName(): Promise<string> {
+    const response = await api.get('/api/memevers/me');
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)